### DISCUSSION TITLE
Add image above profile picture

### DISCUSSION THREAD

**Discussion by dylanschlichting** [alshedivat/al-folio#2019] [Q&A]
I'd like to add an image equal to the text width above the profile picture and subsequent text on the about page. Has anybody been able to do this? If I try placing a figure above the profile pic in about.md it will not render.

**Comment by alshedivat** [comment]
could you share more details of what you are trying to do (ideally, an image or sketch of how you want it to look like) and what exact code changes do not render as expected?

**Reply by dylanschlichting** [reply]
A working example of my website can be found [here](https://dylanschlichting.github.io/). The corresponding source code can be found [here](https://github.com/dylanschlichting/dylanschlichting.github.io). I'd like to insert a picture in the about page directly where the yellow line is showed in the attached figure. This will essentially be a banner, provided the aspect ratio of the image is correct. Making the image equal to the page width is no problem since that is set within the template, but it doesn't render (unsurprisingly) when I do a naive attempt. The other attached figure shows the result of the rendered code. For reference, I am new to html and css so apologies if this is an easy fix. If you have a solution, a stock photo can be used for a minimal working example. 
```
---
layout: about
title: Dylan R. Schlichting
permalink: /
description:

<img style="display: block; margin: 0 auto;" src="../_pages/los_alamos_overlook.jpg" width="800">

profile:
  align: right
  image: prof_pic_NM.jpg
  address: >
    <p>TA-03 200, Los Alamos National Laboratory</p>
    <p>Los Alamos, NM 87545</p>

news: false  # includes a list of news items
selected_papers: true # includes a list of papers marked as "selected={true}"
social: true  # includes social icons at the bottom of the page
---
```
<img width="1007" alt="Screen Shot 2023-12-26 at 5 02 31 PM" src="https://github.com/alshedivat/al-folio/assets/60902283/c2c7068d-14e4-4f51-ba4b-5f00a95d114e">
<img width="1446" alt="Screen Shot 2023-12-26 at 4 57 30 PM" src="https://github.com/alshedivat/al-folio/assets/60902283/b4f5c3ba-bb4d-438f-b4b5-4baa4cb5c3c0">
